app/src/main/java/com/mapbox/maps/testapp/examples/GlobeActivity.kt

```diff
@@ -74,6 +73,7 @@ class GlobeActivity : AppCompatActivity() {
             skyType(SkyType.ATMOSPHERE)
             skyAtmosphereSun(listOf(15.0, 89.5))
           }
+          +projection(ProjectionName.GLOBE)
```

Comment:
`HeatmapLayerActivity` could also be revisited to make use of style DSL loading method